src/main/java/org/jboss/tools/intellij/openshift/ui/project/CreateNewProjectDialog.java

```diff
@@ -33,84 +32,84 @@
+  private class ActiveProjectValidator implements Supplier<ValidationInfo> {
 
-        private final Pattern nameRegex = Pattern.compile(
-          "^[a-z0-9]+(([.-])?[a-z0-9])$");
+    private final Pattern nameRegex = Pattern.compile(
+      "^[a-z0-9]+(?:-[a-z0-9]+){0,64}$");
 
-        @Override
-        public ValidationInfo get() {
-            String activeProject = newProjectTextField.getText();
-            ValidationInfo validation = getValidationInfo(activeProject);
-            // update OK button
-            setOKActionEnabled(validation.okEnabled);
-            return validation;
-        }
+    @Override
+    public ValidationInfo get() {
+      String activeProject = newProjectTextField.getText();
+      ValidationInfo validation = getValidationInfo(activeProject);
+      // update OK button
+      setOKActionEnabled(validation.okEnabled);
+      return validation;
+    }
 
-        private ValidationInfo getValidationInfo(String newProject) {
-            ValidationInfo validation = new ValidationInfo("").withOKEnabled();
-            if (StringUtil.isEmptyOrSpaces(newProject)) {
-                validation = new ValidationInfo("Provide name", newProjectTextField).asWarning();
-            } else if (newProject.length() > 63) {
-                validation = new ValidationInfo("Mustn't have > 63 characters", newProjectTextField).asWarning();
-            } else if (StringUtils.isNumeric(newProject)) {
-                validation = new ValidationInfo("Mustn't contain only numeric characters", newProjectTextField).asWarning();
-            } else if (allProjects.contains(newProject)) {
-                validation = new ValidationInfo("Already exists, choose new name", newProjectTextField).asWarning();
-            } else if (!nameRegex.matcher(newProject).matches()) {
-                validation = new ValidationInfo("Must be alphanumeric and may contain -", newProjectTextField);
-            }
-            return validation;
-        }
+    private ValidationInfo getValidationInfo(String newProject) {
+      ValidationInfo validation = new ValidationInfo("").withOKEnabled();
+      if (StringUtil.isEmptyOrSpaces(newProject)) {
+        validation = new ValidationInfo("Provide name", newProjectTextField).asWarning();
+      } else if (newProject.length() > 63) {
+        validation = new ValidationInfo("Mustn't have > 63 characters", newProjectTextField).asWarning();
+      } else if (newProject.chars().allMatch(Character::isDigit)) {
+        validation = new ValidationInfo("Mustn't contain only numeric characters", newProjectTextField).asWarning();
+      } else if (allProjects.contains(newProject)) {
+        validation = new ValidationInfo("Already exists, choose new name", newProjectTextField).asWarning();
+      } else if (!nameRegex.matcher(newProject).matches()) {
+        validation = new ValidationInfo("Must be alphanumeric and may contain -", newProjectTextField);
```

Comment:
Maybe we should also mention that characters have to be lowercase:
```
Must be lowercase alphanumeric and may contain -"
```

Reply:
fixed by using same terms as kubectl validation